wfe/web-front-end.go

```diff
@@ -78,6 +81,28 @@ func (wfe *WebFrontEndImpl) HandlePaths() {
 
 // Method implementations
 
+func (wfe *WebFrontEndImpl) Index(response http.ResponseWriter, request *http.Request) {
+	// http://golang.org/pkg/net/http/#example_ServeMux_Handle
```

Comment:
By that example I get the impression that the HandleFunc for / also needs to be called last. Can you double check / test?